autonomica-api/test_caching.py

```
@@ -0,0 +1,365 @@
+    loaded_value = new_persistent_cache.get("persistent_key")
+    
+    if loaded_value == "persistent_value":
+        print("✅ Persistent storage working correctly")
+    else:
+        print("❌ Persistent storage failed")
+    
+    # Clean up test file
+    try:
+        os.remove("test_cache.json")
+    except:
+        pass
```

Ruff (0.12.2)

Use `contextlib.suppress(BaseException)` instead of `try`-`except`-`pass`

Replace with `contextlib.suppress(BaseException)`

(SIM105)